src/kit/LogViewer/LogViewer.tsx

```diff
@@ -529,8 +532,20 @@ const LogViewer: React.FC<Props> = ({
     [dateTimeWidth],
   );
 
+  useLayoutEffect(() => {
+    if (height !== undefined) {
+      setLogContainerSize(height);
+    } else if (mainRef.current !== null) {
+      // CSS min-height property doesn't quite work on setting the height of current container as 100%
+      // takes the parent container height, if any, or the available container height
+      mainRef.current.parentElement
+        ? setLogContainerSize(mainRef.current.parentElement.getBoundingClientRect().height)
+        : setLogContainerSize(mainRef.current.getBoundingClientRect().height);
+    }
+  }, [height]);
```

Comment:
Since this does not help with the styling on Determined app, should we still keep it?